data/sles4sap/qe_sap_deployment/qesap_aws.yaml

```diff
@@ -0,0 +1,35 @@
+provider: "%PROVIDER%"
+apiver: 2
+terraform:
+  variables:
+    aws_region: "%REGION%"
+    deployment_name: "%DEPLOYMENTNAME%"
+    os_image: "%QESAP_CLUSTER_OS_VER%"
+    os_owner: "amazon"
```

Comment:
This needs to be a `%VARIABLE%` also. On QAM we would test with ami's owned by amazon, while on Product QA, we would test with a self owned image uploaded by an openQA job.

Keep in mind that my comment is moot if we have a `qesap_aws.yaml` file for QAM and another for Product QA.

Reply:
`qesap_aws.yaml` can be considered dedicated to this test only.
I'm able to find other files using the os_owner variable in the repo, but all of them has a value hard-coded to "amazon" or "self". Is there any example of a file using a template variable for it. I'm mostly interested to check where the template gets the value from.

Reply:
Yes. Here: http://mango.qa.suse.de/tests/5023#step/hana_sr_deployment/71

It's getting the value from the PUBLIC_CLOUD_IMAGE_OWNER setting.

More details in my branch from 2 weeks ago:

- https://github.com/alvarocarvajald/os-autoinst-distri-opensuse/commit/8ee61157dd4bee861f05add0ced8f6966afd6e35#diff-3226f0b86bf0b058a92784f787d19467ebecc5246fbcc5e7750455d3f63734f6R12
- https://github.com/alvarocarvajald/os-autoinst-distri-opensuse/commit/8ee61157dd4bee861f05add0ced8f6966afd6e35#diff-98a44adfecdae80a9b5eab7a2922a193b903d8fa846b355db78783ff85c13b9bR45

Reply:
ok, I was looking that on https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/variables.md

I plan only to quse these qesap_*.yaml for the tests/sles4sap/qesapdeploy/* tests and accept the limitation only to run those tests agains amazon owned images. Does it make sense?

Reply:
Yes, it does.